Runtime/AvatarCreator/UI/Buttons/ButtonElement.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace ReadyPlayerMe.AvatarCreator
+{
+    [RequireComponent(typeof(Button))]
+    public class ButtonElement : MonoBehaviour
+    {
+        [SerializeField] private Button button;
+        [SerializeField] private RawImage rawImage;
+
+        public void AddListener(Action action)
+        {
+            button.onClick.AddListener(action.Invoke);
+        }
+
+        private void OnDestroy()
+        {
+            button.onClick.RemoveAllListeners();
```

Comment:
May this is not necessary but I am not sure.

Reply:
Yeah I'm pretty sure the it's super unlikely that this is necessary it was more of a good practice thing

Reply:
cleaned